src/__test__/components/data-exploration/differential-expression-tool/DiffExprResults.test.jsx

```diff
@@ -241,32 +240,34 @@ describe('DiffExprResults', () => {
 
     // // Wait for side-effect to propagate (properties loading and loaded).
     await waitForActions(withResultStore, [DIFF_EXPR_ORDERING_SET, DIFF_EXPR_LOADING, DIFF_EXPR_LOADED]);
+    const entries = component.find(".ant-table-tbody").children();
 
-    expect(fetchWork).toHaveBeenCalledWith(
```

Comment:
fetchwork is still called only once right? to get all the genes? So maybe you should still have a check for it here